test/evergreen/run_format_configs.sh

```diff
@@ -21,20 +21,59 @@ fi
 
 success=0
 failure=0
+running=0
+# Jobs to execute in parallel (hard coded to 8)
+parallel_jobs=8
+PID=""
+declare -a PID_LIST
 
+# Wait for other format runs.
+wait_for_other_format_runs()
+{
+	for process in ${PID_LIST[@]};do
+		# Remove the process id of the completed runs from the process id array.
+		unset PID_LIST[0]
+		PID_LIST=(${PID_LIST[@]})
+		wait $process
+		exit_status=$?
+		let "running--"
+		config_name=`egrep $process $tmp_file | awk -F ":" '{print $2}' | rev | awk -F "/" '{print $1}' | rev`
+		if [ $exit_status -ne "0" ]; then
+			let "failure++"
+			[ -f WT_TEST_${config_name}/CONFIG ] && cat WT_TEST_${config_name}/CONFIG
+		else
+			let "success++"
+			# Remove database files of successful jobs.
+			[ -d WT_TEST_${config_name} ] && rm -rf WT_TEST_${config_name}
+		fi
+
+		return
```

Comment:
This will only wait for the first process. Shouldn't it check either if any of them finish (they should all have the same parent) each time through?

Reply:
Yes, it waits for the first process to finish from the PID_LIST array because I save the process ID and config mapping saved in a temp file that I need to find the home directory and retain or delete the files.

Reply:
I think it should all for all slots to be used at all times (in case we have long running tasks that just end up at the top of the list).

Reply:
Fixed in 913a1ba0e1fca8768e30cd529b12f8703ab90b27